Jenkinsfile

```diff
@@ -31,26 +34,44 @@ pipeline {
       }
     }
 
-    stage ("Run Integration Tests") {
+    /* we want to avoid running in parallel otherwise we can get gcloud crashed : database is locked
+       working with same environment in parallel */
+   stage ("Run Integration Tests on oss") {
       steps {
         script {
           def tasks = [:]
-          ["oss", "dap"].each { deployment ->
-            tasks["Kubernetes GKE, ${deployment}"] = {
-                sh "./bin/test_integration --docker --${deployment} --gke"
+            tasks["Kubernetes GKE, oss"] = {
+                sh "./bin/test_integration --docker --oss --gke"
             }
-            tasks["Openshift v3.11, ${deployment}"] = {
-                sh "./bin/test_integration --docker --${deployment} --oc311"
+            tasks["Openshift v3.11, oss"] = {
+                sh "./bin/test_integration --docker --oss --oc311"
             }
-            tasks["Openshift v3.10, ${deployment}"] = {
-                sh "./bin/test_integration --docker --${deployment} --oc310"
+            tasks["Openshift v3.10, oss"] = {
+                sh "./bin/test_integration --docker --oss --oc310"
             }
-          }
           parallel tasks
         }
       }
     }
 
+    stage ("Run Integration Tests on dap") {
```

Comment:
can you change `dap` to `DAP` throughout the file? except for `--dap` of course.

Reply:
👍